zapcore/level.go

```diff
@@ -49,6 +49,9 @@ const (
 	PanicLevel
 	// FatalLevel logs a message, then calls os.Exit(1).
 	FatalLevel
+
+	minLevel = DebugLevel
+	maxLevel = FatalLevel
```

Comment:
I'm not always good about remembering, but we should underscore-prefix unexported globals.